### PR TITLE
DKO HOTFIX

### PR DESCRIPTION
**Pull request by webcoderz**
hotfix for DKO, went back to old cudaworker-daemonset (1 worker per gpu node) and dask scheduler deployment set up for the time being until DKO is more stable (restores workers after casualty)

### CODE REVIEW

File: charts/graphistry-helm/templates/notebook/notebook-deployment.yaml [hunk 1]
```diff
@@ -115,6 +115,9 @@ spec:
               name: gak-public
             - mountPath: /home/graphistry/graph-app-kit-private/
               name: gak-private
+              #*WILL REMOVE FOR RELEASE , THIS IS FOR A DATA PROJECT*
```
Comment (lmeyerov) [inline charts/graphistry-helm/templates/notebook/notebook-deployment.yaml:130]: just calling out for review

File: charts/graphistry-helm/templates/notebook/notebook-deployment.yaml [hunk 2]
```diff
@@ -128,6 +131,10 @@ spec:
         - name: gak-private
           persistentVolumeClaim:
             claimName: gak-private
+        ###data pvc *WILL REMOVE FOR RELEASE , THIS IS FOR A DATA PROJECT*
```
Comment (lmeyerov) [inline charts/graphistry-helm/templates/notebook/notebook-deployment.yaml:147]: calling this one out too

File: cd/repo/Chart.yaml [hunk 3]
```diff
@@ -1,9 +1,8 @@
 apiVersion: v2
 name: dask-kubernetes-operator-subchart
 type: application
-version: 2023.1.1
-appVersion: "2022.4.1"
+version: 2022.11.2
 dependencies:
 - name: dask-kubernetes-operator
-  version: 2023.1.1 
+  version: 2022.11.2
```
Comment (lmeyerov) [inline cd/repo/Chart.yaml:7]: are you on 2.40.1 graphistry images (or whatever starts using rapids 23.02)?

**Comment by webcoderz** [comment]
closing and moving PR to cherry picked branch

**Pull request closed without merging**